src/components/common/FeatureBox/FeatureBox.module.scss

```diff
@@ -4,6 +4,7 @@
   border: 1px solid $feature-border-color;
   text-align: center;
   margin-top: 40px;
+  min-height: 9rem;
```

Comment:
Ale jeśli content się zmieni i bedzie większy niż to 9rem, to nadal elementy nie bedą równe. Wykorzystaj to, że dzieci flexa w jednym row są tej samej wysokości, trzeba będzie użyć height: 100% i odpowiednio ustawić paddingi.

Reply:
Poprawione